cscs-checks/prgenv/gpu_direct_acc.py

```diff
@@ -29,14 +29,17 @@ def __init__(self, **kwargs):
             self.num_tasks_per_node = 8
 
         self.sourcepath = 'gpu_direct_acc.F90'
-        self.sanity_patterns = sn.assert_found(r'Result :\s+OK', self.stdout)
+        self.sanity_patterns = sn.all([sn.assert_found(r'GPU with OpenACC',
+                                                       self.stdout),
+                                       sn.assert_found(r'Result :\s+OK',
+                                                       self.stdout)])
         self.maintainers = ['AJ', 'VK']
         self.tags = {'production'}
 
     def setup(self, partition, environ, **job_opts):
-        if environ.name == 'PrgEnv-cray':
+        if 'PrgEnv-cray' in environ.name:
             environ.fflags = '-hacc -hnoomp'
-        elif environ.name == 'PrgEnv-pgi':
+        elif 'PrgEnv-pgi' in environ.name:
```

Comment:
Perhaps, it's better to test if `environ.name.startswith('PrgEnv-pgi')`.

Reply:
Can you also move this file under `cscs-checks/mch/`?

Reply:
done